pandas/tests/plotting/test_style.py

```diff
@@ -0,0 +1,162 @@
+from contextlib import suppress
+
+import pytest
+
+import pandas.util._test_decorators as td
+
+from pandas import Series
+
+with suppress(ImportError):
+    from pandas.plotting._matplotlib.style import get_standard_colors
+
+
+@td.skip_if_no_mpl
```

Comment:
you could just use pytest.importorskip see pandas\tests\plotting\test_converter.py or other test modules testing optional dependencies. e.g. pytables

Reply:
Did that. Please check if this is what you suggested.